mediathread/templates/taxonomy/taxonomy.html

```diff
@@ -114,15 +114,14 @@ <h2>
                         <div class="card-body">
                         <h3>Terms</h3>
                             <div class="row">
-                            <div class="col-md-10">
-                                <input type="text" name="term_name" class="form-control float-left" placeholder="Type new term name here"/>
-                            <a class="btn btn-light create-term-submit float-left" href="#">
-                                <img src="{{STATIC_URL}}img/icons/meth_addselection.png" alt="Create term" />
-                            </a>
-                            </div>
+                                <div class="col-md-10">
```

Comment:
I'm still seeing the diagonal paint of the icons down the screen.

Reply:
Ah! Something is up with my browser. It keeps showing me the correct layout and I have to clear cache to see that it's not correct